### ISSUE TITLE
Unity freezes during asset import if a ButtplugClient is connected

### ISSUE THREAD

**Issue by McArdellje** [buttplugio/buttplug-unity#17]
Issue occured in Unity 2019.4.31f1
I know that it is probably not intended for this library to be used in the editor its self, but I was attempting to do this, and I found that when ButtplugClient is connected and any asset is imported (not just a script, any asset at all), the editor will freeze up completely, permanently loading with a CPU usage of 0%, meaning that it isn't even processing anything, just completely frozen.

**Comment by qdot** [comment]
Yeah I'm honestly not sure if that's possible. We make a lot of assumptions about async runtimes that may not hold in a version of Unity that old.
